exampleCourse/questions/examplesNumberInput/question.html

```diff
@@ -0,0 +1,83 @@
+<pl-question-panel>
+  <p>
+    Questions below were designed to showcase the different features of 
+    <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a>. 
+    The goal of <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a>
+    is to allow users to enter <b>real or complex values</b> to be checked within a specific tolerance.  For comparison, the correct value of $c = {{correct_answers.c_1}}$.
+  </p>
+</pl-question-panel>
+
+<div class="card my-2">
+  <div class="card-body">
+    <pl-question-panel>
+      <p>Within this question, you can observe the default behavior of the <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a> element.
+        In particular, the default input field has help text as a placeholder as well as a question button and is set to use <code>"relabs"</code> to check the tolerance, which is a combination of relative (<code>"rtol"</code>) and absolute (<code>"atol"</code>) tolerances.</p>
+      <p> Solve for the value of $c$ given $c^2 = {{params.a}}^2 + {{params.b}}^2$ </p>
+    </pl-question-panel>
+    <pl-number-input answers_name="c_1" />
+  </div>
+</div>
+
+<div class="card my-2">
+  <div class="card-body">
+    <pl-question-panel>
+      <p>This question modifies the default relative tolerance (of <code>1e-3</code>) and absolute tolerance (of <code>1e-8</code>) underneath <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a> element. Furthermore, we add a label to the front of the input field to specify what the field should receive. </p>
+      <p> Solve for the value of $c$ given $c^2 = {{params.a}}^2 + {{params.b}}^2$ </p>
+    </pl-question-panel>
+    <pl-number-input answers_name="c_2" label = "$c = $" rtol = "1e-3" atol="1e-10"/>
```

Comment:
Missed this in the first round though, but we should update all attributes to used the dashed version (e.g. `answers_name`). @nicknytko definitely realize you didn't write this originally but we should correct this before introducing this to master.

Reply:
I also noticed that some of the elements are not "properly closed" in this example. 

`<pl-number-input ....></pl-number-input>`

@nicknytko can you also make these changes please?